argilla/docs/how_to_guides/query.md

```diff
@@ -122,7 +122,7 @@ You can use the `Filter` class to define the conditions and pass them to the `Da
 
 ## Filter by status
 
-You can filter records based on their status. The status can be `pending`, `draft`, `submitted`, or `discarded`.
+You can filter records based on record or response status. Record status can be `pending` or `completed` and response status can be `pending`, `draft`, `submitted`, or `discarded`.
```

Comment:
I'm unsure if I understand the difference between record and response status. Why/When one or the other?